pop/POPAnimation.mm

```diff
@@ -270,3 +270,43 @@ - (id)pop_animationForKey:(NSString *)key
 }
 
 @end
+
+@implementation POPAnimation (NSCopying)
+
+- (instancetype)copyWithZone:(NSZone *)zone
+{
+  POPAnimation *copy = nil;
+  
+  // Must use [self class] instead of POPAnimation so that subclasses can call this via super.
+  // Since POPAnimation and POPPropertyAnimation are (semi-)concrete, wrap initialization in a try-catch.
+  // If regular initialization fails, use _init.
```

Comment:
Ohhhh, I see what you're saying. I now realize it's not necessary to have the try-catch.

I guess the way to do it would just use `POPAnimation *copy = [[[self class] allocWithZone:zone] init]`

and since there must be an object to call `copy` on, the check is unecessary?